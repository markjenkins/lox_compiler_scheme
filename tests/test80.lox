// from craftinginterpreters/test/block/empty.lox

{} // By itself.

// In a statement.
if (true) {}
if (false) {} else {}

print "ok"; // expect: ok
