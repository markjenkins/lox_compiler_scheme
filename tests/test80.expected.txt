OP_TRUE
OP_JUMP_IF_FALSE 4
OP_POP
OP_JUMP 1
OP_POP
OP_FALSE
OP_JUMP_IF_FALSE 4
OP_POP
OP_JUMP 1
OP_POP
OP_CONSTANT "ok"
OP_PRINT
OP_RETURN
